macros/Cargo.toml

```diff
@@ -26,4 +26,4 @@ syn = { version = "2", features = ["full", "extra-traits"] }
 [dev-dependencies]
 maplit = "1"
 pretty_assertions = "1"
-rstest = { version = "0.19", default-features = false }
+rstest = { version = "0.23", default-features = false }
```

Comment:
Unless we rely on a feature in 0.23, we don't need this. Cargo will resolve to the latest anyway and there's no use on over-specifying the dependencies in case users have conflicting requirements.